CMakeLists.txt

```diff
@@ -173,6 +173,10 @@ file(GLOB S2N_SRC
 add_library(${PROJECT_NAME} ${S2N_HEADERS} ${S2N_SRC})
 set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE C)
 
+# As long as the ABI isn't stable, set the SO version to 0unstable
+set_target_properties(${CMAKE_PROJECT_NAME} PROPERTIES VERSION 1.0.0)
```

Comment:
As s2n is not yet at 1.0, maybe:
```
set_target_properties(${CMAKE_PROJECT_NAME} PROPERTIES VERSION ${MAJORVER}.${MINORVER})
set_target_properties(${CMAKE_PROJECT_NAME} PROPERTIES SOVERSION ${MAJORVER})
```

Reply:
I was basically doing what AWS is using in other projects like ```aws-c-common``` or ```aws-checksums```. Both used version ```1.0.0``` and the SO version ```unstable0```.

But, of course, I can change it anyway you want.

Reply:
What version would you suggest for a project that hasn't had a 1.0.0 release yet? unstable0 for both?

Reply:
The library version number is that important, so you can choose any value you want. But the SO version should have something that indicates whether the API is stable or not.